File: 백준/Bronze/2566. 최댓값/최댓값.c
#include<stdio.h>
int main(void){
    int arr;
    int max = 0;
    int x = 0;
    int y = 0;
    
    for(int i = 0; i < 9; i++){
        for(int j = 0; j < 9; j++){
            scanf("%d",&arr);
            if(max < arr){
                max = arr;
                x = i;
                y = j;
            }
        }
    }
    printf("%d\n",max);
    printf("%d %d\n",x+1,y+1);
    
    return 0;
}

File: 백준/Bronze/2566. 최댓값/최댓값.py
arr = []
max = 0
for i in range(9):
    arr.append(list(map(int, input().split())))
    for j in arr[i]:
        if max <= j:
            max = j

print(max)
for i in range(9):
    for j in range(9):
        if arr[i][j] == max:
            print(i + 1, j + 1)
            break